### ISSUE TITLE
The recognition result cannot be displayed in NVDA version alpha-21371

### ISSUE THREAD

**Issue by cary-rowen** [alekssamos/cloudvision#5]
NVDA: alpha-21371,376edfcf
Add-on Version:  2.0.3.7

Quickly press NVDA+CTRL+I twice. After the recognition result is returned, an error sound will be heard, and NVDA is forced to restart.

error log:
INFO - __main__ (10:15:59.742) - MainThread (16148):
Starting NVDA version alpha-21371,376edfcf
INFO - core.main (10:15:59.888) - MainThread (16148):
Config dir: C:\Users\manch\AppData\Roaming\nvda
INFO - config.ConfigManager._loadConfig (10:15:59.888) - MainThread (16148):
Loading config: C:\Users\manch\AppData\Roaming\nvda\nvda.ini
INFO - core.main (10:16:00.001) - MainThread (16148):
Using Windows version 10.0.19042 workstation
INFO - core.main (10:16:00.001) - MainThread (16148):
Using Python version 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:01:55) [MSC v.1900 32 bit (Intel)]
INFO - core.main (10:16:00.001) - MainThread (16148):
Using comtypes version 1.1.7
INFO - core.main (10:16:00.002) - MainThread (16148):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - synthDriverHandler.setSynth (10:16:00.504) - MainThread (16148):
Loaded synthDriver vocalizer_expressive2
INFO - core.main (10:16:00.504) - MainThread (16148):
Using wx version 4.0.3 msw (phoenix) wxWidgets 3.0.5 with six version 1.12.0
INFO - brailleInput.initialize (10:16:00.505) - MainThread (16148):
Braille input initialized
INFO - braille.initialize (10:16:00.506) - MainThread (16148):
Using liblouis version 3.15.0
INFO - braille.initialize (10:16:00.509) - MainThread (16148):
Using pySerial version 3.4
INFO - braille.BrailleHandler.setDisplayByName (10:16:00.513) - MainThread (16148):
Loaded braille display driver noBraille, current display has 0 cells.
INFO - core.main (10:16:00.721) - MainThread (16148):
Java Access Bridge support initialized
INFO - _UIAHandler.UIAHandler.MTAThreadFunc (10:16:00.732) - _UIAHandler.UIAHandler.MTAThread (6816):
UIAutomation: IUIAutomation6
DEBUGWARNING - inputCore.InputManager.loadLocaleGestureMap (10:16:00.822) - MainThread (16148):
No locale gesture map for language zh_CN
DEBUGWARNING - touchHandler.touchSupported (10:16:00.858) - MainThread (16148):
No touch devices found
DEBUGWARNING - abstractEngine.AbstractEngineHandler.getEngineList (10:16:00.929) - MainThread (16148):
Engine 'captcha' doesn't pass the check, excluding from list
DEBUGWARNING - abstractEngine.AbstractEngineHandler.getEngineList (10:16:00.931) - MainThread (16148):
Engine 'sougouOCR' doesn't pass the check, excluding from list
DEBUGWARNING - abstractEngine.AbstractEngineHandler.getEngineList (10:16:00.938) - MainThread (16148):
Engine 'machineLearning' doesn't pass the check, excluding from list
IO - external:globalPlugins.fixime.GuessComposition.get_code_by_name (10:16:00.949) - MainThread (16148):
controlconverted to 17
IO - external:globalPlugins.fixime.GuessComposition.get_code_by_name (10:16:00.949) - MainThread (16148):
numpadenterconverted to 13
IO - external:globalPlugins.fixime.GuessComposition.get_code_by_name (10:16:00.949) - MainThread (16148):
enterconverted to 13
INFO - external:globalPlugins.fixime.GlobalPlugin.refreshLayoutString (10:16:00.954) - MainThread (16148):
2052
INFO - external:globalPlugins.fixime.GlobalPlugin.refreshLayoutString (10:16:00.954) - MainThread (16148):
00000804
IO - inputCore.InputManager.executeGesture (10:16:00.958) - winInputHook (15064):
Input: kb(laptop):shift+windows+space
INFO - external:globalPlugins.fixime.GlobalPlugin.refreshLayoutString (10:16:00.961) - MainThread (16148):
2052
INFO - external:globalPlugins.fixime.GlobalPlugin.refreshLayoutString (10:16:00.961) - MainThread (16148):
00000804
IO - inputCore.InputManager.executeGesture (10:16:00.962) - winInputHook (15064):
Input: kb(laptop):windows+space
INFO - external:globalPlugins.fixime.GlobalPlugin.refreshLayoutString (10:16:00.965) - MainThread (16148):
2052
INFO - external:globalPlugins.fixime.GlobalPlugin.refreshLayoutString (10:16:00.965) - MainThread (16148):
00000804
INFO - core.main (10:16:01.128) - MainThread (16148):
NVDA initialized
IO - speech.speak (10:16:01.293) - MainThread (16148):
Speaking ['空格', EndUtteranceCommand()]
IO - speech.speak (10:16:01.295) - MainThread (16148):
Speaking ['空格', EndUtteranceCommand()]
IO - speech.speak (10:16:01.386) - MainThread (16148):
Speaking ['放大', '按钮']
IO - inputCore.InputManager.executeGesture (10:16:01.885) - winInputHook (15064):
Input: kb(laptop):escape
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IA2States (10:16:01.904) - MainThread (16148):
could not get IAccessible2 states
Traceback (most recent call last):
  File "NVDAObjects\IAccessible\__init__.pyc", line 1633, in _get_IA2States
  File "comtypesMonkeyPatches.pyc", line 26, in __call__
_ctypes.COMError: (-2147417848, '被调用的对象已与其客户端断开连接。', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IA2Attributes (10:16:01.905) - MainThread (16148):
IAccessibleObject.attributes COMError (-2147417848, '被调用的对象已与其客户端断开连接。', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (10:16:01.906) - MainThread (16148):
accRole failed: (-2147417848, '被调用的对象已与其客户端断开连接。', (None, None, None, 0, None))
IO - speech.speak (10:16:01.990) - MainThread (16148):
Speaking ['桌面', '列表']
IO - speech.speak (10:16:01.994) - MainThread (16148):
Speaking ['NVDA']
IO - inputCore.InputManager.executeGesture (10:16:02.333) - winInputHook (15064):
Input: kb(laptop):alt+f4
IO - speech.speak (10:16:02.482) - MainThread (16148):
Speaking ['关闭 Windows', '对话框', '关闭所有应用，然后关闭电脑。']
IO - speech.speak (10:16:02.488) - MainThread (16148):
Speaking ['希望计算机做什么(W)?', '组合框', '关机', '已折叠', 'Alt+w']
IO - inputCore.InputManager.executeGesture (10:16:02.830) - winInputHook (15064):
Input: kb(laptop):alt+f4
IO - speech.speak (10:16:02.882) - MainThread (16148):
Speaking ['桌面', '列表']
IO - speech.speak (10:16:02.884) - MainThread (16148):
Speaking ['NVDA']
IO - inputCore.InputManager.executeGesture (10:16:03.137) - winInputHook (15064):
Input: kb(laptop):alt+tab
IO - speech.speak (10:16:03.211) - MainThread (16148):
Speaking ['亲情永恒']
IO - speech.speak (10:16:03.442) - MainThread (16148):
Speaking ['亲情永恒']
IO - speech.speak (10:16:03.553) - MainThread (16148):
Speaking ['输入', '编辑框', '多行', '空白']
IO - inputCore.InputManager.executeGesture (10:16:03.746) - winInputHook (15064):
Input: kb(laptop):alt+f4
IO - speech.speak (10:16:03.823) - MainThread (16148):
Speaking ['桌面', '列表']
IO - speech.speak (10:16:03.831) - MainThread (16148):
Speaking ['NVDA']
IO - inputCore.InputManager.executeGesture (10:16:04.903) - winInputHook (15064):
Input: kb(laptop):control+NVDA+i
IO - speech.speak (10:16:04.907) - MainThread (16148):
Speaking ['识别查看对象']
DEBUGWARNING - Python warning (10:16:04.908) - MainThread (16148):
C:\Users\manch\AppData\Roaming\nvda\addons\CloudVision\globalPlugins\CloudVision\__init__.py:198: wxPyDeprecationWarning: Call to deprecated item EmptyBitmap. Use :class:`wx.Bitmap` instead
  bmp = wx.EmptyBitmap(width, height)
IO - inputCore.InputManager.executeGesture (10:16:05.049) - winInputHook (15064):
Input: kb(laptop):control+NVDA+i
INFO - external:globalPlugins.fixime.patchIMESupport.inputLangChangeNotify (10:16:17.933) - Dummy-5 (2988):
threadID16148
hkl:134481924
layoutString微软拼音

INFO - external:globalPlugins.fixime.patchIMESupport.patchedHICM (10:16:17.951) - MainThread (16148):
oldFlags
1
newFlags:
1025
lcid
2052
INFO - external:globalPlugins.fixime.patchIMESupport.inputLangChangeNotify (10:16:17.991) - Dummy-5 (2988):
threadID392
hkl:134481924
layoutString微软拼音

IO - speech.speak (10:16:18.091) - MainThread (16148):
Speaking ['识别结果']
IO - inputCore.InputManager.executeGesture (10:16:19.741) - winInputHook (15064):
Input: kb(laptop):tab
IO - inputCore.InputManager.executeGesture (10:16:20.094) - winInputHook (15064):
Input: kb(laptop):escape
CRITICAL - watchdog._crashHandler (10:16:18.282) - Dummy-6 (392):
NVDA crashed! Minidump written to C:\Users\manch\AppData\Local\Temp\nvda.log\..\nvda_crash.dmp
INFO - watchdog._crashHandler (10:16:20.650) - Dummy-6 (392):
Listing stacks for Python threads:
Python stack for thread 392 (Dummy-6):
  File "watchdog.pyc", line 213, in _crashHandler
  File "watchdog.pyc", line 63, in getFormattedStacksForAllThreads

Python stack for thread 15732 (virtualBuffers.MSHTML.VirtualBuffer.loadBuffer):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 926, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "virtualBuffers\__init__.pyc", line 444, in _loadBuffer

Python stack for thread 13792 (watchdog):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 926, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "watchdog.pyc", line 120, in _watcher

Python stack for thread 15064 (winInputHook):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 926, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "winInputHook.pyc", line 79, in hookThreadFunc

Python stack for thread 6816 (_UIAHandler.UIAHandler.MTAThread):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 926, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "_UIAHandler.pyc", line 315, in MTAThreadFunc
  File "queue.pyc", line 170, in get
  File "threading.pyc", line 296, in wait

Python stack for thread 12476 (Thread-1):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 926, in _bootstrap_inner
  File "C:\Users\manch\AppData\Roaming\nvda\addons\vocalizer_expressive2_driver\synthDrivers\vocalizer_expressive2\_vocalizer.py", line 34, in run
    instance, inText = self._bgQueue.get()
  File "queue.pyc", line 170, in get
  File "threading.pyc", line 296, in wait

Python stack for thread 16148 (MainThread):
  File "nvda.pyw", line 247, in <module>
  File "core.pyc", line 568, in main
  File "wx\core.pyc", line 2134, in MainLoop
  File "gui\__init__.pyc", line 1062, in Notify
  File "core.pyc", line 544, in run
  File "baseObject.pyc", line 159, in invalidateCaches
  File "comtypesMonkeyPatches.pyc", line 105, in newCpbDel
  File "comtypes\__init__.pyc", line 918, in __del__
  File "comtypes\__init__.pyc", line 1172, in Release
  File "comtypesMonkeyPatches.pyc", line 26, in __call__

INFO - watchdog._crashHandler (10:16:20.650) - Dummy-6 (392):
Restarting due to crash
IO - speech.speak (10:16:20.697) - MainThread (16148):
Speaking ['正在加载文档...']

**Comment by alekssamos** [comment]
Ok, thanks, I figured out what the problem is and will fix it shortly.

**Comment by alekssamos** [comment]
Hello, @cary-rowen  .
The restart issue is already in NVDA. To check it out, Open NVDA menu, service, the python console and run these two commands:

```python3
import ui

ui. browseableMessage("some text", "title")

```

Report an error in the" ui.browseableMessage " function in nvaccess.

**Comment by cary-rowen** [comment]
OK, Thanks, I have reproduced this issue.
Digression:

Is it possible to allow CloudVision to obtain the image data it wants to recognize from the clipboard?
